flexmeasures/data/schemas/sensors.py

```diff
@@ -242,11 +242,22 @@ def _deserialize(
 
         elif isinstance(value, str):
             try:
+                try:
+                    float(value)
+                    return value
+                except ValueError:
+                    pass
                 return ur.Quantity(value).to(self.to_unit)
             except DimensionalityError as e:
                 raise FMValidationError(
                     f"Cannot convert value `{value}` to '{self.to_unit}'"
                 ) from e
+        elif (
+            isinstance(value, Sensor)
+            or isinstance(value, int)
+            or isinstance(value, float)
+        ):
+            return value
```

Comment:
@Flix6x, after these changes, [this](https://github.com/FlexMeasures/flexmeasures/blob/7f01658cf413ffdf31dac4ba0ede83a2a922a0b2/flexmeasures/data/schemas/tests/test_scheduling.py#L242) test is failing.